config/jobs/k8s-spot-rescheduler/k8s-spot-rescheduler-postsubmits.yaml

```diff
@@ -36,9 +36,9 @@ postsubmits:
         containers:
           - <<: *container_template_large
             args:
-              - touch .env; # No config necessary
+              - ./configure
```

Comment:
You'll need to add `&&` after this else the shell will try to execute `./configure make docker...` which won't work ☹️